src/components/carrusel/CarrucelComponent.jsx

```diff
@@ -18,6 +18,9 @@ const CarrucelComponent = () => {
       .get(`${process.env.REACT_APP_API_URL}/cuisine-categories`)
```

Comment:
As an enhancement: assign the env value to a constant and that one would be better here.